packages/block-library/src/button/save.js

```diff
@@ -46,7 +45,6 @@ export default function save( { attributes, className } ) {
 		borderProps.className,
 		typographyProps.className,
 		{
-			[ `has-text-align-${ textAlign }` ]: textAlign,
```

Comment:
This could be considered a breaking change if themes or extenders rely on this class.